<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FiscalReportView</class>
 <widget class="QWidget" name="FiscalReportView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FiscalReportView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Месец на описа:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="monthCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Януари</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Февруари</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Март</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Април</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Май</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Юни</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Юли</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Август</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Септември</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Октомври</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ноември</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Декември</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="yearSpin">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="suffix">
        <string> г.</string>
       </property>
       <property name="minimum">
        <number>1900</number>
       </property>
       <property name="maximum">
        <number>3000</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="generateFiscalDescr">
       <property name="text">
        <string>Генерирай отчетен документ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="deleteReceipt">
       <property name="text">
        <string>Изтрий фискален опис</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
